src/platform/silabs/rs911x/rsi_ble_config.h

```diff
@@ -256,4 +256,4 @@ typedef struct rsi_ble_s
     rsi_ble_att_list_t att_rec_list[NO_OF_VAL_ATT];
 } rsi_ble_t;
 
-#endif
+#endif
```

Comment:
coding standard/styler should fix this. No need to commit this

Reply:
ok I will fix in coding style patch

Reply:
un-resolving since it's still present here..